androidComposePrototype/src/androidTest/java/com/genesys/cloud/messenger/uitest/test/ComposePrototypeUITest.kt

```diff
@@ -155,22 +165,13 @@ class ComposePrototypeUITest : BaseTests() {
             }
             apiHelper.sendConnectOrDisconnect(conversationInfo, false, true)
         } else AssertionError("Agent did not answer conversation.")
-        messenger {
-            enterCommand(byeText)
-            waitForClosed()
-        }
+        apiHelper.disconnectAllConversations()
     }
 
     @Test
-    fun verifyAutoStart() {
-        opening {
-            verifyPageIsVisible()
-            enterDeploymentID()
-            selectView(testBedViewText)
-        }
-        messenger {
-            verifyPageIsVisible()
-        }
+    // Adjusting the test name to force this test to run first
+    fun testAAVerifyAutoStart() {
```

Comment:
is it a typo with double `A` here?

Reply:
I adjusted the name of the test to force it to run first. (tests running in alphabetic order)

Reply:
Are digits run before letters? We can make it more clear by naming it `test1VerifyAutostart`, but it is not critical for me.